resources/resource/dates.go

```diff
@@ -44,19 +44,19 @@ func (d *Dates) UpdateDateAndLastmodIfAfter(in Dated) {
 }
 
 // IsFuture returns whether the argument represents the future.
-func IsFuture(d Dated) bool {
+func IsFuture(buildTime time.Time, d Dated) bool {
```

Comment:
One goal of having a global override was to minimise the changes needed. This means that we only want to change the import from `time.Now()` to `htime.Now()` in the relevant places.